src/main/resources/org/csanchez/jenkins/plugins/kubernetes/PodTemplate/config.jelly

```diff
@@ -17,6 +17,14 @@
     <f:textbox/>
   </f:entry>
 
+  <f:entry title="${%Usage}" help="/help/system-config/master-slave/usage.html">
```

Comment:
this is defaulting to `use this node as much as possible` and should be the other one